docs/data/system/experimental-api/extend-sx-prop/ChangeTheBehaviorSxProp.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { createTheme, ThemeProvider, Box, Stack } from '@mui/system';
+
+// Retain type safety.
+declare module '@mui/system' {
+  interface Shape {
+    sm: number;
+    md: number;
+    lg: number;
+  }
+}
+
+const theme = createTheme({
+  unstable_sxConfig: {
+    // You can now use the background key in sx
+    // by providing direct values from the palette
+    borderRadius: {
+      themeKey: 'shape',
+    },
+  },
+  shape: {
+    sm: 4,
+    md: 8,
+    lg: 12,
+  },
+});
+
+export default function ExtendSxProp() {
+  return (
+    <ThemeProvider theme={theme}>
+      <Stack direction="row" gap={1}>
```

Comment:
This would improve the inline-preview's SNR.

```suggestion
    <Stack direction="row" gap={1}>
      <ThemeProvider theme={theme}>
```